### ISSUE TITLE
eggnog-mapper package not built for python3

### ISSUE THREAD

**Issue by 0xaf1f** [bioconda/bioconda-recipes#26364]
This package still only exists for python2.

```
$ conda install eggnog-mapper
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: \ 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                             

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - eggnog-mapper -> python[version='2.7.*|<3|>=2.7,<2.8.0a0']

Your python: python=3.6

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.
```

**Comment by 0xaf1f** [comment]
The upstream development repository's master branch is still for Python, but they have a Python 3 branch. See https://github.com/eggnogdb/eggnog-mapper/issues/260

**Comment by bgruening** [comment]
Yes, we need to wait until eggnog has a stable release for python3. Can we close this issue?

**Comment by 0xaf1f** [comment]
Thanks - But first, let me ping them to see if they will cut a Python 3 release. Maybe this can be resolved quickly.

**Comment by 0xaf1f** [comment]
Here's the response I got: https://github.com/eggnogdb/eggnog-mapper/issues/260#issuecomment-769209584
Not sure whether it means it'll happen soon or that they've been working on it and it'll still take time. Close if you wish.

**Comment by 0xaf1f** [comment]
Looks like a new release as a Python3 version has come out

**Comment by Tierhon** [comment]
The 2.1.0 version seems to support Python3 now. I'd be great if eggnog-mapper was updated to that version.

**Comment by bgruening** [comment]
Hi,

you can update the package by updating this file: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/eggnog-mapper/meta.yaml

Let us know if you need help,
Bjoern

**Comment by druvus** [comment]
eggnog-mapper was fixed  and updated in #28296 so it is now python3 compatible.
